### ISSUE TITLE
imagePullSecret nindent error?

### ISSUE THREAD

**Issue by RyanMcCarthy852** [airbytehq/airbyte#16413]
## Environment
- **Airbyte version**:  0.40.3
- **Airbyte charts version**: 0.45.1
- **OS Version / Instance**: Windows 10
- **Deployment**: OpenShift
- **Step where error happened**: Deploy 

## Current Behavior
Fails to deploy with the following in values.yaml:
```
global: 
    imagePullSecrets: 
      -  name: secretName
```
due to yaml parsing error I think stemming from [webapp deployment.yaml](https://github.com/airbytehq/airbyte/blob/master/charts/airbyte-webapp/templates/deployment.yaml):
```
      {{- if .Values.global.imagePullSecrets }}
      imagePullSecrets:
        {{- range .Values.global.imagePullSecrets }}
          {{- printf "- name: %s" .name | nindent 2 }}
        {{- end }}
      {{- end }}
```

I think nindent just needs to be bumped to 8. I made this change and repackaged charts and that's what worked and I see my secret in k8s appropriately afterward. 

New to helm please inform me if I'm misinformed on something here. 

Believe this applies to these 4 deployment.yaml files: https://github.com/airbytehq/airbyte/pull/15967/files

But then applied that block to podsweeper/temporal anything else where it may be desirable to specify imagePullSecrets for an internal registry as this change doesn't exist in those yet. 

## Steps to Reproduce
1. add imagePullSecret to global
2. kubectl upgrade/install airbyte
3. chart returns a failure to parse yaml error

## Are you willing to submit a PR?
I cannot without significant annoying conversations within current company :(

**Comment by bdashrad** [comment]
Looks like this was fixed by #16131

**Comment by shaoye** [comment]
> Looks like this was fixed by #16131

thx, it's not released yet

**Comment by marcosmarxm** [comment]
Closed by #16131
